src/Translator.zig

```diff
@@ -735,7 +781,8 @@ fn transType(t: *Translator, scope: *Scope, qt: QualType, source_loc: TokenIndex
         .void => return ZigTag.type.create(t.arena, "anyopaque"),
         .bool => return ZigTag.type.create(t.arena, "bool"),
         .int => |int_ty| switch (int_ty) {
-            .char => return ZigTag.type.create(t.arena, "c_char"),
+            //.char => return ZigTag.type.create(t.arena, "c_char"), // TODO: why is this different than clang translate-c?
+            .char => return ZigTag.type.create(t.arena, "u8"),
```

Comment:
It's because of strings, `char` in C is usually signed while Zig string literals are `u8`.

Changing this doesn't need to block this PR since currently we are trying to reach feature parity with the Clang version.